<!-- Launch file to detect aruco markers -->

<launch>

    <group ns="saf">

    <node name="aruco_detector_realsense" pkg="irob_vision_support"
            type="aruco_detector_realsense.py" output="screen" >

                <remap from="image_markers" to="vision/image_markers" />
                <remap from="markers" to="vision/markers" />
    </node>
    </group>
</launch>
